### ISSUE TITLE
Quartz build requires SVN installed

### ISSUE THREAD

**Issue by vpavic** [quartz-scheduler/quartz#66]
On a machine without SVN installed project build fails with the following error:

```
[ERROR] Failed to execute goal org.terracotta:maven-forge-plugin:1.1.9:manifest (create-manifest) on project quartz: Execution create-manifest of goal org.terracotta:maven-forge-plugin:1.1.9:manifest failed: Execute failed: java.io.IOException: Cannot run program "svn": error=2, No such file or directory -> [Help 1]
```

IMO the build should not depend on SVN since the project has moved to Github.

**Comment by jhouserizer** [comment]
That's silly indeed.

**Comment by zemian** [comment]
This can be fixed by removing `org.terracotta:maven-forge-plugin` plugin in `quartz/pom.xml`. Would anyone objects if we remove this outdated enforcement?

**Comment by zemian** [comment]
This is now fixed by #290
